pkg/kubelet/dockertools/docker.go

```diff
@@ -211,6 +211,7 @@ func (p throttledDockerPuller) IsImagePresent(name string) (bool, error) {
 	return p.puller.IsImagePresent(name)
 }
 
+// TODO (random-liu) Almost never used, should we remove this?
```

Comment:
This is used by `GetKubeletDockerContainers`. Do you want to rewrite the function?